src/docs/design/project-identity.mdx

```diff
@@ -0,0 +1,153 @@
+    avatar={<PxblueSmallAltIcon style={{ width: 48, height: 48 }} />}
+    minCardHeight={'unset'}
+/>
+
+## PX Blue is Flexible
+
+PX Blue is not prescriptive — just as we build on top of Material Design principles, you too can build on top of PX Blue principles to make your products better for your target audience. There are many gray areas and gaps in PX Blue where you are free to make changes and tweaks.
+
+<ImageGrid
+    regularWidth
+    images={[ConsumerIndustrial]}
+    caption={[
+        `An example of two projects, both designed to control devices, but with different targeted audience.`,
+        `1: A consumer-faced product. End user might be homeowners, small business owners, etc. The project is designed to delight the users.`,
+        `2: Industrial product. End user might be asset managers, facility assets managers, etc. The project is designed to  convey a secure and efficient picture for equipment management.`,
+    ]}
+/>
+
+We fully expect you to establish your own local design system for your project that extends PX Blue and adds your own flair / style / identity. Depending on your use case, you may want your design system to be more flashy, visual, relaxed, friendly, etc. Depending on where your application will reside, you may also need to change aspects of the design system related to accessibility (such as larger buttons for touch-screens) or legal-requirements in your target markets.
+
+<ImageGrid
+    regularWidth
+    images={[StyleChoice]}
+    caption={[
+        `You may tune styles for your application away from the PX Blue baseline style, as part of your local design system. In this example, the app bar (#1), the input form field (#2) and the buttons (#3) are changed.`,
+    ]}
+/>
+
+While establishing your local product identity, remember that PX Blue is here to help establish consistency, so you do not want to go so far down the road of customization that this is lost. Below, we go into details about some common areas of customization as well as some areas that should be avoided.
+
+## Common Things to Change
+
+There are some areas where we strongly encourage you to make changes and establish your own brand identity.
+
+Keep in mind that once you've established a local design system, you must make sure that you apply it consistently everywhere. You should use it in your packaging, your website, your email templates, the application, etc.
+
+### Theme
+
+PX Blue [offers two themes](/style/themes): Blue theme and Blue Dark theme. Ideally, an application should support both themes, but some applications might be more usable depending on the environment and context of the usage.
+
+By default, we use `#007bc1` as our primary theme color. But this is the baseline color and may vary from projects to projects. You may change the primary theme color, given that you carefully design it to avoid inflicting with the color convention ([see below](#color-convention)).
```

Comment:
```suggestion
By default, we use `#007bc1` as our primary theme color, but this can vary from project to project. You may change the primary theme color, given that you carefully design it to avoid conflicting with the color convention ([see below](#color-convention)).
```